packages/ckeditor5-cloud-services/src/token/token.ts

```diff
@@ -112,14 +114,29 @@ export default class Token extends /* #__PURE__ */ ObservableMixin() {
 				}
 
 				return this as InitializedToken;
+			} )
+			.catch( err => {
+				/**
+				 * TODO
+				 *
+				 * @error token-refresh-failed
+				 */
+				console.warn( 'token-refresh-failed: TODO' );
```

Comment:
Warn and retry or retry and re-throw the error?

Reply:
Decision: Both. Warn to let people know what's going on. Throw for people who manually call `refreshToken()` and they need a properly rejected promise.